app/views/tasks/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<h2>This is edit page</h2>
```

Comment:
You've got almost exactly the same code here to render a form as you do in `new`. You should dry this up with a view partial.